pyproject.toml

```diff
@@ -37,6 +37,12 @@ docs = [
   "mkdocs-jupyter>=0.24.1",
   "nbstripout>=0.6.1",
 ]
+mattersim = [
```

Comment:
Let's just call it `extra` or `calc` or `forcefields` and consider including other forcefields in the future